cnd/cnd.debugger.common2/src/org/netbeans/modules/cnd/debugger/common2/debugger/Autos.java

```diff
@@ -22,30 +22,29 @@
 import java.util.Collections;
 import java.util.Set;
 import javax.swing.text.StyledDocument;
-import org.netbeans.modules.cnd.spi.model.services.AutosProvider;
-import org.openide.util.Lookup;
 
 public final class Autos {
```

Comment:
Just out of curiosity - what are `Autos`? Some kind of variable?

Reply:
`auto`  is a [keyword in C89](https://en.wikipedia.org/wiki/C_(programming_language)) (seldomly used, though).  It was later [enhanced in C++11](https://docs.microsoft.com/en-us/cpp/cpp/auto-cpp?view=msvc-170) and In C++14 they aded the so called [type deduction with braced initializers](https://docs.microsoft.com/en-us/cpp/cpp/auto-cpp?view=msvc-170#type-deduction-with-braced-initializers-c14), as horrible as that looks! :-D